tt_llk_blackhole/llk_lib/llk_math_eltwise_binary_sfpu_params.h

```diff
@@ -0,0 +1,62 @@
+// SPDX-FileCopyrightText: © 2025 Tenstorrent AI ULC
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+#include "llk_math_eltwise_binary_sfpu.h"
+#include "llk_sfpu_types.h"
+
+template <bool APPROXIMATE, class F, class... ARGS>
+inline void _llk_math_eltwise_binary_sfpu_params_(F&& sfpu_func, uint dst_index0, uint dst_index1, int vector_mode = (int)VectorMode::RC, ARGS&&... args)
+{
+    uint dst_index  = (dst_index0 <= dst_index1) ? dst_index0 : dst_index1;
+    uint dst_offset = (dst_index0 > dst_index1) ? dst_index0 - dst_index1 : dst_index1 - dst_index0;
```

Comment:
```suggestion
    const uint dst_index  = std::min(dst_index0, dst_index1);
    const uint dst_offset = std::max(dst_index0, dst_index1) - dst_index;
```
Don't forget to `#include <algorithm>`.